pkg/db/columnhandler.go

```diff
@@ -147,6 +147,39 @@ func (ch *ColumnHandler) ColumnNamesCsv(onlyWriteable bool) string {
 	return buffer.String()
 }
 
+// TODO
```

Comment:
If we go with immutable objects (which would be good to have automatically also the history and auditing-problem addressed) we don't need updates actually.